### DISCUSSION TITLE
Samsung Compatibility Question

### DISCUSSION THREAD

**Discussion by joeyoropesa-dev** [reiryuki/Miui-Core-Magisk-Module#35] [Q&A] [answered]
Does MiuiCore module is allowed to work on any model and manufacture or it can only work on specified devices. If it can only work for specified devices and manufacture (Xiaomi), could some port for Samsung be done. Thanks!

**Comment by reiryuki** [comment, marked as answer]
Please READ the Requirements!

**Reply by joeyoropesa-dev** [reply]
> Please READ the Requirements!

Nice, I didn't saw in Requirements part any mentions about that device model and manufacture needs to be by Xiaomi so I believe I should be fine with installing this module. What you think since it's not fixed on specific models and Xiaomi manufacture? Thank you so much for quick responses! ✨🫂
